tests/run/sourcePosition.scala

```diff
@@ -0,0 +1,22 @@
+import dotty.source.Position._
+
+object Test {
+  def main(args: Array[String]): Unit = {
+    println(thisSource)
+    println(thisLine)
+    println()
+    println(implicitly[SourcePath])
+    println(implicitly[LineNumber])
+    println()
+    printSourcePath()
+    printPosition()
+    println()
+    printPosition(
+
+    )
+  }
+
+  def printPosition()(implicit pos: LineNumber): Unit = println(pos)
+  def printSourcePath()(implicit src: SourcePath): Unit = println(src)
```

Comment:
What would happen if you call for `SourcePath` or `LineNumber` in a named argument? I guess it's worth adding a test for that. And also for the constructors.

Reply:
They work as expected.